src/components/exchange/ConfirmExchangeButton.js

```diff
@@ -157,7 +157,9 @@ export default function ConfirmExchangeButton({
             label={label}
             loading={loading}
             onLongPress={
-              insufficientLiquidity
+              loading
+                ? () => null
+                : insufficientLiquidity
```

Comment:
Is `() => null` just meant to be a noop? Or is there some importance around returning null?

If not, it might be better to use `Function.prototype` because, I think, it will point to the same object each render giving us a cached value for memoization.

Reply:
yes only noop, the button was enabled when you were pressing on a "Fetching quote" state, I'm gonna add it at top of file

Reply:
we usually add this somewhere in the file and then use `const NOOP = () => undefined;`